src/Combobox/index.js

```diff
@@ -17,14 +18,26 @@ export const VALID_ICON_NAMES = iconSelection.icons.map(
   (icon) => icon.properties.name,
 );
 
+/**
+ * @param {Object} item Combobox.{Action|Item|Heading} component
+ * @returns {Boolean} true if the item is a Combobox.Action
+ */
+export const isAction = (item) => {
+  let result = false;
+  if (item && item.props) {
+    result = "label" in item.props;
+  }
+  return result;
+};
+
 /**
  * @param {Object} component a Combobox.Item or Combobox.Heading component
- * @returns {Boolean} true if the item is a selectable Combobox.Item
+ * @returns {Boolean} true if the item is a selectable Combobox.Item or Action
  */
 export const isSelectable = (component) => {
```

Comment:
"selectable" means selectable via keyboard as part of the options list of the combobox